content/W2016/W16-media-stephanie.md

```diff
@@ -0,0 +1,16 @@
+Title: Stephanie Morillo Talk
+Date: 2016-03-22
+Category: Media
+Tags: talks
+Slug: stephanie-morillo-video-2016
+Author: Fatema Boxwala
+Summary: The amazing Stephanie Morillo gives a talk entitled Circuit Breaker
```

Comment:
Can you maybe say what the talk was about, and end the summary with a period?

The Sarah talk summary was "Sarah Sharp, Linux kernel developer, visits Waterloo to give a talk about getting involved with the Linux kernel and open source."